config/metac.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: metac.openebs.io/v1alpha1
+kind: GenericController
+metadata:
+  name: sync-openebs
+  namespace: openebs-test
+spec:
+  updateAny: true
+  watch:
+    apiVersion: dao.mayadata.io/v1alpha1
+    resource: openebses
+  attachments:
+  - apiVersion: dao.mayadata.io/v1alpha1
+    resource: openebses
+    updateStrategy:
```

Comment:
as per our latest conversations, we dont need update strategy for openebs resource.
However, we need to add various other resources as attachments e.g. the resources that are going to be created due to openebs resource.